src/constants/labels.json

```
@@ -188,7 +188,12 @@
     "repeatWords": "Wörter wiederholen",
     "repeatNow": "Jetzt wiederholen",
     "wordsToRepeat": "Wörter zum Wiederholen bereit",
-    "yourLearningProgress": "Dein Lernfortschritt"
+    "yourLearningProgress": "Dein Lernfortschritt",
+    "chart": {
+      "new_": "Neue",
+      "learned": "Gelernte",
```

I think it would be better to also include the word `Wörter` here since it adds more context and the order might be different for other languages (if we should ever decide to translate the app).
```suggestion
      "learnedWords": "Gelernte Wörter",
```

---

We need "gelernte" and "wörter" as two separte strings, as this is how the library works. I think is is easier to just define them here as it is and avoid using split functions.